### PR TITLE
xml code-style

### PR DESCRIPTION
**Pull request by brianteeman**
update the xml to follow the joomla codestyle for xml files

@quy can you review please

### CODE REVIEW

File: src/components/com_weblinks/views/categories/tmpl/default.xml [hunk 1]
```diff
@@ -25,136 +27,160 @@
...
-				description="COM_WEBLINKS_SHOW_EMPTY_CATEGORIES_DESC"
-				useglobal="true"
-			>
-				<option value="0">JHIDE</option>
-				<option value="1">JSHOW</option>
-			</field>
-
-			<field name="show_subcat_desc" type="list"
-			description="JGLOBAL_SHOW_SUBCATEGORIES_DESCRIPTION_DESC"
+		<field name="show_category_title" type="list"
```
Comment (zero-24) [inline src/components/com_weblinks/views/categories/tmpl/default.xml]: please move name and type to a new line.

File: src/components/com_weblinks/views/category/tmpl/default.xml [hunk 2]
```diff
@@ -72,124 +85,141 @@
...
+				default=""
 				useglobal="true"
-			>
+				>
 				<option value="hide">JHIDE</option>
 				<option value="1">JSHOW</option>
 			</field>
 
-			<field	name="show_pagination_limit" type="list"
+			<field	n
```
Comment (zero-24) [inline src/components/com_weblinks/views/category/tmpl/default.xml]: looks like there is a typo ;)

**Comment by zero-24** [comment]
most things looks goof to me. I have left some inline comments.

**Comment by brianteeman** [comment]
Thanks @zero-24 fixed those

File: src/components/com_weblinks/views/categories/tmpl/default.xml [hunk 3]
```diff
@@ -98,15 +98,18 @@
 		</field>
 	</fieldset>
 
-	<fieldset name="category" label="JGLOBAL_CATEGORY_OPTIONS">
+	<fieldset 
+	name="category" label="JGLOBAL_CATEGORY_OPTIONS">
```
Comment (zero-24) [inline src/components/com_weblinks/views/categories/tmpl/default.xml]: did you this change for a reason? IMO this line can be reverted and than it is ready to merge. IMO

**Comment by zero-24** [comment]
Thanks @brianteeman

**Comment by brianteeman** [comment]
thanks

**Comment by Quy** [comment]
Sorry to be late. Found some issues:

https://github.com/brianteeman/weblinks/blob/f290ad3e234998d3695dee48dafe73b90feed354/src/components/com_weblinks/models/forms/weblink.xml#L14

https://github.com/brianteeman/weblinks/blob/f290ad3e234998d3695dee48dafe73b90feed354/src/components/com_weblinks/views/category/tmpl/default.xml#L15
https://github.com/brianteeman/weblinks/blob/f290ad3e234998d3695dee48dafe73b90feed354/src/components/com_weblinks/views/category/tmpl/default.xml#L212

https://github.com/brianteeman/weblinks/blob/f290ad3e234998d3695dee48dafe73b90feed354/src/modules/mod_weblinks/mod_weblinks.xml#L88

https://github.com/brianteeman/weblinks/blob/f290ad3e234998d3695dee48dafe73b90feed354/src/plugins/search/weblinks/weblinks.xml#L24-L25

**Comment by brianteeman** [comment]
it has been merged :(

so please create a PR

**Comment by zero-24** [comment]
I have fixed them now with direct commits. thanks.

**Pull request merged**